docs/tutorials/dependencies.html.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Danger + Dependencies
+subtitle: Plugin creation
+layout: guide_js
+order: 0
+---
+
+## Before we get started
+
+This guide continues after "[Getting Started][started]" - so you should have seen Danger comment on your PRs.
+
+## Keeping on top of your dependencies
+
+Building pretty-much anything in the node ecosystem involves using using external dependencies. In an ideal situation you want to use as few dependencies as possible, and get the most use out of them. Remember that you are shipping your dependencies too, so you are responsible for them to your end-users.
+
+The numerical scale of dependencies can make it tough to feel like you own your entire stack. So let's try use Danger to give us more insight into changes related to our dependencies.
+
+## Lockfiles
+
+The simplest rule, which we can evolve, is that any time your `package.json` changes you probably want a change to the [`yarn.lock`][lockfile] or [`shrinkwrap.json`][shrinkwrap] file. Yes, not every change to the `package.json` represents a dependency update but we're starting simple. You start off your `Dangerfile` like this:
+
+```js
+import { danger, fail, warn } from "danger"
+import includes from "lodash.includes"
+
+const hasPackageChanges = includes(danger.git.modified_files, "package.json")
+const hasLockfileChanges = includes(danger.git.modified_files, "yarn.lock")
+if (hasPackageChanges && !hasLockfileChanges) {
+  warn("There are package.json changes with no corresponding lockfile changes")
+}
+```
+
+This uses `lodash`'s include function to see if `danger.git.modified_files` includes the package, but not the lockfile.
```

Comment:
"This uses lodash's `_.includes()` function" could be a bit clearer around what include means.